apps/protocol-frontend/src/components/HomeShell.tsx

```diff
@@ -86,7 +86,9 @@ const HomeShell = () => {
                   bgGradient="linear(to-l, #7928CA, #FF0080)"
                   bgClip="text"
                 >
-                  {userData?.name || userDataByAddress?.name}
+                  {userData?.name ||
+                    userDataByAddress?.name ||
+                    displayAddress(userData?.address)}
```

Comment:
I think we should abstract this or statement into a function. I can imagine us adding more logic such as ENS

Reply:
Good call. Maybe we can do the fallbacks like this:
User name > ens > truncated address 

Will make a function we can reuse or adjust